packages/venia-ui/lib/components/RichContent/PageBuilder/ContentTypes/Block/block.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import CmsBlock from '../../../../CmsBlock/block';
```

Comment:
Given that `CmsBlock` does nothing, why doesn't this component use `RichContent` directly?

Reply:
See https://github.com/magento/pwa-studio/pull/1872#discussion_r336003557